### PR TITLE
Create OpenITI AOCP Phase Two.md

### PR DESCRIPTION
**Pull request by pverkind**
This is a placeholder page for a description of Phase Two of the project. It is based on the page for Phase One, with two paragraphs describing Phase Two from the grant proposal.

### CODE REVIEW

File: _projects/OpenITI AOCP Phase Two.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: "*Open Islamicate Texts Initiative Arabic-script OCR Catalyst Project* (*OpenITI AOCP*) Phase Two"
+layout: page
+image: /assets/images/main-images/Isfahan_Lotfollah_mosque_ceiling_symmetric_narrow_border.png
```
Comment (mabarber92) [inline _projects/OpenITI AOCP Phase Two.md:4]: This should be 'banner' now

**Comment by mabarber92** [comment]
I'm merging and then making a couple of changes. Because preview thumbs weren't working properly, yml headers now use the 'banner' field for the banner image and 'image' for the thumb that would appear if you shared a link (e.g. to social media).

**Comment by pverkind** [comment]
Hi Mathew, you might have to make the same changes to the Phase One page
(where I copied the header from)

Best,

Peter

**Comment by mabarber92** [comment]
Ah. Thank you for flagging this. Will quickly work on a fix.

**Pull request merged**